Classes/Domain/Model/MetadataSubentry.php

```diff
@@ -0,0 +1,179 @@
+<?php
+
+/**
+ * (c) Kitodo. Key to digital objects e.V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo and TYPO3 projects.
+ *
+ * @license GNU General Public License version 3 or later.
+ * For the full copyright and license information, please read the
+ * LICENSE.txt file that was distributed with this source code.
+ */
+
+namespace Kitodo\Dlf\Domain\Model;
+
+/**
+ * This specifies a way how a metadatum (``tx_dlf_metadata``) may be encoded in a specific data format (``tx_dlf_format``).
+ *
+ * For instance, the title of a document may be obtained from either the MODS
+ * title field, or from the TEIHDR caption. This is modeled as two ``tx_dlf_metadaformat``
+ * that refer to the same ``tx_dlf_metadata`` but different ``tx_dlf_format``.
+ *
+ * This contains the xpath expressions on the model 'Metadata'.
```

Comment:
This comment seems out-of-date or just copied from `Classes/Domain/Model/MetadataFormat.php`.